src/fragments/lib/geo/js/geofences.mdx

```diff
@@ -14,7 +14,12 @@ To add a geofence management component to your map, you can use the [amplify-geo
 Install the necessary dependencies with the following command:
 
 ```bash
-npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react
+npm install maplibre-gl maplibre-gl-js-amplify aws-amplify @aws-amplify/ui-react-geo
+```
+
+To use `withAuthenticator` as in the example below, install the necessary dependency with the following command:
```

Comment:
If the example uses `withAuthenticator`, then we should include _@aws-amplify/ui-react_ in the above install script right?

Reply:
Yes, makes sense to add it in the same line above as we include all dependencies used in the example.